.github/workflows/linux.yml

```
@@ -697,6 +698,13 @@ jobs:
             make -C bld V=1 pytest-ci
           fi
 
+      - name: 'randcurl'
+        if: ${{ contains(matrix.build.install_steps, 'randcurl') }}
+        run: |
+          mkdir run
+          cd run
+          ../.github/scripts/randcurl.pl 60 /home/runner/work/curl/curl/bld/src/curl
```

```suggestion
          ../.github/scripts/randcurl.pl 60 ../bld/src/curl
```
(to avoid hardcoding GHA paths)

---

thanks!